war/pom.xml

```diff
@@ -587,17 +599,42 @@ THE SOFTWARE.
     <profile>
         <id>node-classifier-linux</id>
         <activation><os><family>Linux</family><arch>amd64</arch></os></activation>
-        <properties><node.download.file>node-v${node.version}-linux-x64.tar.gz</node.download.file></properties>
+		<properties>
+			<node.download.file>node-v${node.version}-linux-x64.tar.gz</node.download.file>
+			<node.download.classifier></node.download.classifier>
+		</properties>
```

Comment:
Yes, I had to fix this for x64 windows. The node v4 thing mucked with frontend-plugin's downloads; it needs a classifier to put the download in a different place locally.